identifiers/country-cz/README.md

```diff
@@ -0,0 +1,8 @@
+# OCD Division Identifier
+
+This folder list the following administrative divisions of the Czech Republic :
+
+*  Electoral districts [ref](https://www.senat.cz/senat/volby/hledani/obvody.php?ke_dni=30.1.2020&O=12) as **ed**.
+*  Region [ref](https://www.psp.cz/sqw/organy2.sqw?kr=1) as **region**.
```

Comment:
Since what we're representing with `region` is the admin area 2 regions of the Czech Republic, I think we should probably just reference those directly.

If electoral law changes again to change the set of regions then separate electoral districts should be added rather than deprecating these.

Reply:
Done.